src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -451,10 +452,13 @@ object EntityClient {
       override def entitiesUpdatedSince(
           dateTime: ZonedDateTime,
           startEntry: Int,
-          maxEntries: Int = 1000
+          maxEntries: Int = 1000,
+          potentialEndDate: Option[ZonedDateTime] = None
       ): F[Seq[Entity]] = {
         val dateString = dateTime.format(dateFormatter)
-        val queryParams = Map("date" -> dateString, "max" -> maxEntries, "start" -> startEntry)
+        val endDateParams =
```

Comment:
1. Do you think that it's worth checking that the end date is equal to or after the `dateTime`?
2. Do you think it's worth changing the `dateTime` to something like `dateTimeSince` or `sinceDateTime`?

Reply:
1. No, if someone is daft enough to do that then there's no help for them.
2. Yeah ok, `sinceDateTime` is fine.